images/gitlab/tests/main.tf

```diff
@@ -0,0 +1,98 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digests" {
+  description = "The image digest to run tests over."
+  type = object({
+    kas = string
+  })
+}
+
+variable "namespace" {
+  description = "The namespace to install Gitlab in."
+}
```

Comment:
When would a caller need to specify this?  I'd just synthesize this all the time.

Reply:
oh we don't use this, let's kill this

Reply:
yes, i use it to create the helm release in a different namespace every time. `namespace        = local.namespace`

Reply:
That's the local, this is a variable.

Reply:
Basically, unless we have `var.namespace` then this is dead.  I'd prefer the local and drop this either way.

Reply:
Okay 👍!